### DISCUSSION TITLE
「はじめてつくるWebアプリケーション」の紹介記事を書きたい

### DISCUSSION THREAD

**Discussion by emorima** [rubima/magazine.rubyist.net#405] [記事を書きたい]
https://github.com/rubima/magazine.rubyist.net/pull/399#issuecomment-1515542729 でお話した内容で、
「はじめてつくるWebアプリケーション」の紹介記事を
著者の 3名 @emorima @yada-ita @cobachie で書きたいです！

**Comment by kakutani** [comment]
0063号でやれます??  @emorima @yada-ITA @cobachie

**Comment by neko314** [comment]
@emorima @yada-ITA @cobachie 

お待たせしましたー
記事公開しました🙂
https://magazine.rubyist.net/articles/0063/0063-1st-steps-into-rails.html

**Comment by neko314** [comment]
記事を執筆いただいてありがとうございました。
公開したのでこのディスカッションはクローズしますね。
